static/js/map/map.pokemon.js

```diff
@@ -630,3 +648,17 @@ function sendPokemonNotification(pokemon) {
     notificationData.cp_multiplier = pokemon.cp_multiplier
     notifiedPokemonData[pokemon.encounter_id] = notificationData
 }
+
+function getSizeDisplay(size){
+       return getSizes()[size] ?? null
+}
+
+function getSizes(){
```

Comment:
I don't believe this needs to be a function.
Can you create a static array near the top of the utils.pokemon.js file. Something like the following line:
const pokemonSizes = ['Unknown', 'XXS', 'XS', 'M', 'XL', 'XXL']

The "getSizeDisplay" function will need to be updated for this change.